test/sql.test

```diff
@@ -41,17 +41,27 @@ namespace eval ::qcode::test {
     test sql_sort-1.0 {sql_sort} -setup {
     } -body {
         sql_sort name email user_code 
-    } -cleanup {} -result {name,email,user_code}
+    } -cleanup {} -result {"name","email","user_code"}
```

Comment:
what happens when you call something like `qc::sql_sort column_a DESC NULLS LAST`?
Is this pattern ever used for MLA?